test/e2e/basic_integration_test.go

```diff
@@ -0,0 +1,34 @@
+package e2e
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("When testing basic cluster creation [basic]", func() {
+	BeforeEach(func() {
+		osType := strings.ToLower(os.Getenv("OS"))
+		Expect(osType).ToNot(Equal(""))
+		validateGlobals(specName)
+
+		// We need to override clusterctl apply log folder to avoid getting our credentials exposed.
+		clusterctlLogFolder = filepath.Join(os.TempDir(), "clusters", bootstrapClusterProxy.GetName())
+	})
+
+	It("Should create a workload cluster", func() {
+		By("Fetching cluster configuration")
+		numberOfWorkers = int(*e2eConfig.GetInt32PtrVariable("WORKER_MACHINE_COUNT"))
+		numberOfControlplane = int(*e2eConfig.GetInt32PtrVariable("CONTROL_PLANE_MACHINE_COUNT"))
```

Comment:
where are these vars being used?

Reply:
I copied them over from the integration test, they are not used there either. Only place where they're read seems to be in pivoting_based_feature_test.go.

Reply:
Removed them.

Reply:
Are not they used in `createTargetCluster` ?

Reply:
seems that will get from e2e_suite https://github.com/metal3-io/cluster-api-provider-metal3/blob/660431812eea1a522e37940eddd1ef600095bd5f/test/e2e/e2e_suite_test.go#L117